decorators.rb

```diff
@@ -0,0 +1,13 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @component.correct_name.upcase
```

Comment:
- [ ] Using `.upcase` will return all the letter in capital but we dont need that here, here we have to use `capitalize`, which will return only the first letter capital. For ex:-

```
name="adam"
name.upcase         # "ADAM"
name.capitalize      # "Adam"
```

Reply:
@Deepakdanger Thanks for taking the time and pointing out the issue 😇. I will fix the issue right away 😊